db/seeds.rb

```diff
@@ -72,7 +72,7 @@ def all_old_guides
       phase:           "beta",
       description:     "Description",
       update_type:     "minor",
-      body:            object[:body],
+      body:            object[:body].gsub(/\n/, "\r\n"),
```

Comment:
No, rails app saves it with \r\n. This fixes the problem, tested locally
On Wed, 25 Nov 2015 at 16:29, Andrew Vos <notifications@github.com> wrote:

> In db/seeds.rb
> <https://github.com/alphagov/service-manual-publisher/pull/80#discussion_r45887379>
> :
>
> > @@ -72,7 +72,7 @@ def all_old_guides
> >        phase:           "beta",
> >        description:     "Description",
> >        update_type:     "minor",
> > -      body:            object[:body],
> > +      body:            object[:body].gsub(/\n/, "\r\n"),
>
> Shouldn't this be .gsub /\r\n?/, "\n"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/service-manual-publisher/pull/80/files#r45887379>
> .
>